lib/conjur_client.rb

```diff
@@ -4,7 +4,15 @@
 class ConjurClient
   class << self
     def api
-      @@api
+      ConjurClient.new.api
+    end
+
+    def v4_host_factory_id
+      "#{account}:host_factory:#{policy}/pcf-apps"
```

Comment:
ack. the second pcf is actually `#{policy}` too.

Reply:
Fixed